### PR TITLE
Merge working branch

### PR DESCRIPTION
**Pull request by davidauza-engineer**
Let us know your comments about this branch

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey folks excellent work with this project you almost past on your first review  :1st_place_medal: 

## Sign up button
Please add the signup button and corresponding test within the right corner Thanks! 

-  [x] also there should a be an input field with  **type='tel'**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            <span class="menu-item"><a href="#">mint</a></span>
+            <img class="menu-logo" src="images/ic_quickbooks.png" alt="Mint logo">
+            <span class="menu-item"><a href="#">quickbooks</a></span>
+            <img class="menu-logo" src="images/ic_turbotax.png" alt="Mint logo">
+            <span class="menu-item"><a href="#">turbotax</a></span>
+        </div>
+        <!-- Menu Container End -->  
+        <!-- Form Container Start -->  
+        <div class="form-container">    
+            <h1 class="signin-form">Sign In</h1>
```
Comment (Ispirett) [inline index.html]: Please replace this with **Create an Intuit account**

**Review by Ispirett — CHANGES_REQUESTED** [review]

**Comment by davidauza-engineer** [comment]
All changes done :)

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Nicely Done team 👏
I'll give this project a 👍. Go on and merge to master. Good luck with the next one.
